src/lib/isMobilePhone.js

```diff
@@ -150,7 +150,7 @@ const phones = {
   'th-TH': /^(\+66|66|0)\d{9}$/,
   'tr-TR': /^(\+?90|0)?5\d{9}$/,
   'tk-TM': /^(\+993|993|8)\d{8}$/,
-  'uk-UA': /^(\+?38|8)?0\d{9}$/,
+  'uk-UA': /^(\+?38)?0(50|63|66|67|68|73|91|92|93|94|95|96|97|98|99)\d{7}$/,
```

Comment:
```suggestion
  'uk-UA': /^(\+?38)?0(50|6[36-8]|73|9[1-9])\d{7}$/,
```
Minor suggestion to reduce the length of the expression

Reply:
I changed and added two new mobile carrier codes

Reply:
National commission for state regulation in the spheres of electronic communications, radio frequency spectrum and the provision of postal communication services
Results of NKEK meetings from March 6, 2024 [code 75](https://nkrzi.gov.ua/index.php?r=site/index&pg=99&id=2959&language=uk)
Results of the NKEK meeting of February 28, 2024 [code 77](https://nkrzi.gov.ua/index.php?r=site/index&pg=99&id=2953&language=uk)

Reply:
I'm not able to reach https://nkrzi.gov.ua/index.php?r=site/index&pg=99&id=2953&language=uk nor https://nkrzi.gov.ua/
Do you have a different website?

Reply:
https://nkrzi.gov.ua/ is an official site, it may not be accessible from other parts of Ukraine. If necessary, I can attach an image of these solutions from the official website.

Information about this is also available on the official website and on the official data page of mobile operators.
[code 75](https://www.vodafone.ua/news/u-vodafone-zyavitsya-shche-odin-kod-075)
[code 77](https://www.facebook.com/kyivstar/posts/pfbid02LXfUrbjZxAAdPWP98L9JJaU8vdGXnhoQBYXEH7zyg5nVCoPYzTQh8sHTQHs9yTAl)

Reply:
Thanks for linking to the mobile operators. I'll try using a VPN later for the government website and approve afterwards

Reply:
@WikiRik hello, did you manage to view?

Reply:
Yes, thanks for the reminder. With a VPN I was able to view the pages